pkg/apis/cluster/v1alpha1/types.go

```diff
@@ -278,6 +278,10 @@ type LocalSecretReference struct {
 const (
 	// ClusterConditionReady means the cluster is healthy and ready to accept workloads.
 	ClusterConditionReady = "Ready"
+
+	// ClusterConditionIncompleteAPIEnablements means the cluster's APIEnablements are incomplete.
+	// This condition usually occurs when the APIEnablements are not successfully collected.
+	ClusterConditionIncompleteAPIEnablements = "IncompleteAPIEnablements"
```

Comment:
We should try to avoid `double-negative` confusion. Let `true` represent normal, and `false` represent abnormal.